src/ocm/components/clusterConfiguration/staticIp/data/formDataFromInfraEnvField.ts

```diff
@@ -46,6 +46,14 @@ const findFirstRealEthernetInterface = (
   ) as unknown as NmstateEthernetInterface | undefined;
 };
 
+const findFirstVlanEthernetInterface = (
+  interfaces: NmstateInterface[],
+): NmstateEthernetInterface | undefined => {
```

Comment:
should return NmstateVlanInterface

Reply:
return interfaces.find((currentInterface) => isVlanInterface(currentInterface)) as unknown as
    | NmstateEthernetInterface
    | undefined;